lib/cog/relay/relays.ex

```diff
@@ -181,20 +152,28 @@ defmodule Cog.Relay.Relays do
   # If `config` exists in the database (by name), retrieves the
   # database record. If not, installs the bundle and returns the
   # database record.
-  defp lookup_or_install(%{"name" => name, "version" => version} = config) do
+  defp lookup_or_install(%{"name" => name, "version" => version} = config, internal) do
     case Repo.get_by(Bundle, name: name) do
       %Bundle{version: ^version}=bundle ->
         {:ok, bundle}
       %Bundle{version: installed_version} ->
         Logger.error("Error! Unable to install bundle #{inspect name} because version #{installed_version} already installed")
         {:error, name}
       nil ->
-        Logger.info("Installing bundle: #{inspect name}")
-        case Cog.Bundle.Install.install_bundle(%{name: name, version: version, config_file: config}) do
-          {:ok, bundle} ->
-            {:ok, bundle}
-          {:error, error} ->
-            Logger.error("Error! Unable to install bundle #{inspect name}: #{inspect error}")
+        case internal do
+          # This is the embedded bundle
+          true ->
```

Comment:
Since we have the entire bundle loaded here, could we check `bundle.name == Cog.embedded_bundle`, instead of passing in the `internal` variable? Or is this going to be used for other non-embedded bundle stuff eventually?